packages/terra-form-select/src/terra-dev-site/doc/form-select/UpgradeGuide.doc.jsx

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable import/no-webpack-loader-syntax, import/first, import/extensions */
+import React from 'react';
+import DocTemplate from 'terra-doc-template';
+import UpgradeGuide from '../../../../docs/UPGRADEGUIDE.md';
```

Comment:
I kept this as the doc template to keep the upgrade guide in the docs folder so it might be easier for users to find, if anyone has a different preference I can move the guide into this folder.